includes/snmp.inc.php

```diff
@@ -264,6 +282,29 @@ function snmp_get($device, $oid, $options = null, $mib = null, $mibdir = null)
 }//end snmp_get()
 
 
+function snmp_get_next($device, $oid, $options = null, $mib = null, $mibdir = null)
```

Comment:
It is not possible to get the sub-oid of each subsequent radio without making assumptions.
snmp_get_next allows me to walk along the tree for the previously discovered radio count.

Also there are some oids which use index 0 for IP10 but index 127 for IP20C, they may use other indexes for other variants but each variant appears to only use one index so get_next will find it.

Ideally discovery should be provided for each major ( and possibly some minor varients ) eg a discovery for IP10, a different discovery for IP20C, E & S with possibly another one again for the N, etc variants as these provide multiple hot failover options which may not even be polled correctly.

That doesn't seem to fit the LibreNMS model though so I've tried to cover as many bases as possible.

Is it preferred to not use get_next?